doc-changelog/action.yml

```
@@ -253,8 +267,8 @@ runs:
         towncrier create -c "${{ env.CLEAN_TITLE }}" $fragment
 
         # Configure git username & email
-        git config user.name 'pyansys-ci-bot'
-        git config user.email '92810346+pyansys-ci-bot@users.noreply.github.com'
+        git config user.name ${{ env.BOT_USER }}
+        git config user.email ${{ env.BOT_EMAIL }}
```

Echoing incoming @RobPasMue here, is there a need for using `{{ env.BOT_USER }}` (resp. BOT_EMAIL) instead of `{{ inputs.bot-user }}` (resp. bot-email) ?

---

Oops missed this one! 
```suggestion
        git config user.name ${{ inputs.bot-user }}
        git config user.email ${{ inputs.bot-email }}
```